docs/porting.md

```diff
@@ -156,9 +156,49 @@ after importing a module that does not support the GIL.
     free-threaded builds. See [the docs on setting up CI](ci.md) for advice on
     how to build projects that depend on Cython.
 
+=== "Rust"
+    If you use the CPython C API via [PyO3](https://pyo3.rs), then you
+    can follow the [PyO3 Guide
+    section](https://pyo3.rs/v0.23.1/free-threading.html) on supporting
```

Comment:
Maybe you want to link to `latest` instead? I assume significant changes are still coming to that doc.